public/app/features/explore/Logs/Logs.tsx

```diff
@@ -137,6 +137,16 @@ const DEDUP_OPTIONS = [
   LogsDedupStrategy.signature,
 ];
 
+export const visualisationTypeKey = 'grafana.explore.logs.visualisationType';
+
+const getDefaultVisualisationType = (): LogsVisualisationType => {
+  const visualisationType = window.localStorage.getItem(visualisationTypeKey);
```

Comment:
I think we tend to use the `store.get` and `store.set` just as an abstraction layer here. Would be good to be consistent with e.g. https://github.com/grafana/grafana/blob/main/public/app/features/explore/Logs/Logs.tsx#L147-L150